net: lib: azure_iot_hub: Add API to reset DPS information (#6162)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * Added handling of MQTT ping failures and MQTT input failures.
  * Updated the API version used in MQTT connection to Azure IoT Hub to 2020-09-30.

>   * Added function :c:func:`azure_iot_hub_dps_reset` to reset DPS information.

Reviewer: ```suggestion
  * Added the :c:func:`azure_iot_hub_dps_reset` function for resetting the DPS information.
```

Author: Fixed
